routing/probability_bimodal.go

```diff
@@ -67,27 +69,38 @@ func (p *probabilityBimodal) getLocalPairProbability(now time.Time,
 // on the liquidity distribution in the LN.
 func (p *probabilityBimodal) getLiquidityProbability(now time.Time,
 	results NodeResults, toNode route.Vertex, amt lnwire.MilliSatoshi,
-	cap btcutil.Amount) float64 {
-
-	// All the quantities for the algo are expressed in Msat.
-	capacity := lnwire.NewMSatFromSatoshis(cap)
+	capacity lnwire.MilliSatoshi) float64 {
 
 	// Set success and failure amounts.
-	// TODO: include time decay of information.
+
+	// We know that we can send zero satoshis.
 	successAmount := lnwire.MilliSatoshi(0)
+	// We know that we cannot send the full capacity.
 	failAmount := capacity
 
 	result, ok := results[toNode]
 	if ok {
 		if !result.FailTime.IsZero() {
-			failAmount = result.FailAmt
+			// Apply time decay for the amount we can not send.
+			failAmount = cannotSend(
+				result.FailAmt,
```

Comment:
Folding here seems to be a bit wasteful, can pack it into 2 lines or so

Reply:
Changed that :+1: